resources/assets/lib/beatmapsets/beatmapset-json.ts

```diff
@@ -3,6 +3,7 @@
 
 import GenreJson from 'interfaces/genre-json';
 import LanguageJson from 'interfaces/language-json';
+import BeatmapJson from '../interfaces/beatmap-json';
```

Comment:
this must've been automated because it's weirdly different compared to other imports right above it 🙃